express-post-json/index.js

```diff
@@ -0,0 +1,21 @@
+const express = require('express');
+const app = express();
+app.use(express.json());
+
+let nextId = 1;
+const grades = {};
+const gradesArr = [];
+
+app.get('/api/grades', (req, res) => {
+  res.json(gradesArr);
+});
+
+app.post('/api/grades', (req, res) => {
+  grades[nextId] = req.body;
+  grades[nextId].id = nextId;
+  gradesArr.push(grades[nextId]);
```

Comment:
Instead of pushing each new grade into your array, generate the array on-demand in your GET route 👍 

This will make deleting and updating grades _much_ easier.